packages/formstr-app/src/containers/CreateForm/components/QuestionsList/style.ts

```diff
@@ -25,13 +25,13 @@ export default styled.div`
   
   .desktop-add-btn {
     position: sticky;
-    ${MEDIA_QUERY_MOBILE_TABLET} {
+    ${MEDIA_QUERY_MOBILE} {
```

Comment:
why? tablets should also open mobile views no?

Reply:
no, ideally most of the web handles mobile, tablet and desktop UI separately

Reply:
yes , i meant in our case the tablet view should be closer to the mobile view, just going by how it looks on tablet screens